torchvision/prototype/datasets/_builtin/coco.py

```diff
@@ -129,6 +159,7 @@ def _decode_captions_ann(self, anns: List[Dict[str, Any]], image_meta: Dict[str,
     _ANN_DECODERS = OrderedDict(
         [
             ("instances", _decode_instances_anns),
+            ("person_keypoints", _decode_person_keypoints_anns),
```

Comment:
This is unrelated to this PR, but I would maybe revisit if we want to have dataset to have a varying return type depending on arguments passed to it.
It might be better to have different flavors of the dataset to be different dataset classes.

Reply:
Yeah, that is a design choice that we should address. cc @NicolasHug